tests/devicetree_test.py

```diff
@@ -32,7 +33,7 @@ def test_resolve_device(self):
         dev1_label = "dev1_label"
         dev1_uuid = "1234-56-7890"
         fmt1 = get_format("ext4", label=dev1_label, uuid=dev1_uuid)
-        dev1 = StorageDevice("dev1", exists=True, fmt=fmt1)
+        dev1 = StorageDevice("dev1", exists=True, fmt=fmt1, size=Size("2 MiB"))
```

Comment:
It would be better to use `fmt1.min_size` here in case we decide to change it in the future.

Reply:
Fixed.